aws/resource_aws_launch_template.go

```diff
@@ -335,9 +335,20 @@ func resourceAwsLaunchTemplate() *schema.Resource {
 							Optional: true,
 						},
 						"ipv4_addresses": {
-							Type:     schema.TypeSet,
```

Comment:
Unfortunately going from `schema.TypeSet` to `schema.TypeList` is a breaking change since we have no way to consistently migrate existing Terraform configurations. Can you please revert this?